consensus/service/src/api/client_api_service.rs

```diff
@@ -13,7 +13,10 @@ use crate::{
 use grpcio::{RpcContext, RpcStatus, UnarySink};
 use mc_attest_api::attest::Message;
 use mc_attest_enclave_api::ClientSession;
-use mc_common::{logger::Logger, LruCache};
+use mc_common::{
+    logger::{slog::warn, Logger},
+    LruCache,
+};
```

Comment:
:wrench: If the suggestion for the mutex below is acceptable then this will need to be reverted
```suggestion
use mc_common::{logger::Logger, LruCache};
```